### DISCUSSION TITLE
Unable to create new Plug and Play project in testnet

### DISCUSSION THREAD

**Discussion by chenwainuo** [Web3Auth/Web3Auth#684] [Help]
<img width="445" alt="image" src="https://user-images.githubusercontent.com/83727873/194170226-5dd7dca1-497e-4916-ad06-d5715c744894.png">

unable to create new project

receiving: Error: Torus Message Signature: User denied message signature.

**Comment by Felix-Asante** [comment]
I had similar issues , i fixed it by switching to a different browser. in my case i use safari

**Comment by YZhenY** [comment]
Our dashboard is also facing issues due to ropsten being down, we're working on fixing things. Track updates here https://github.com/orgs/Web3Auth/discussions/685
